web/src/cypress/integration/pagination.ts

```diff
@@ -80,6 +80,10 @@ function testPaginating(
         cy.get('body').should('contain', names[itemsPerPage + i])
 
       cy.pageSearch(nameSubstr.slice(0, -1))
+      if (label !== 'Users') {
+        cy.get('body').should('contain', 'No results')
+        cy.pageSearch(nameSubstr)
+      }
```

Comment:
I think this is an actual bug with our tests- we don't support searching by partial substring at the moment so they will always return no results, though I think the intent of the test was to make sure changing the search bar string brings you back to the first page. I think we could re-simulate this by resetting the search bar to `''` and checking that the first test item is contained within the list:

```
cy.pageSearch('')
cy.get('button[data-cy="back-button"]').should('be.disabled')
cy.get('body').should('contain', `Pagination Test ${nameSubstr} 0000`)
```